test/metabase/query_processor/middleware/fetch_source_query_test.clj

```diff
@@ -71,7 +72,24 @@
                   :filter       [:between
                                  [:field "date" {:base-type :type/Date}]
                                  "2015-01-01"
-                                 "2015-02-01"]}))))))))
+                                 "2015-02-01"]})))))))
+  (testing "respects `enable-nested-queries` server setting"
+    (mt/with-temp* [Card [{card-id :id} {:dataset_query (mt/mbql-query venues)}]]
```

Comment:
nit: can just use `with-temp` if you're only creating one temporary model

Reply:
I always use the `with-temp*` so that if i need to add another it doesn't change the syntax. I think `with-temp` should just be ignored